.github/actions/verify-token/src/main.ts

```diff
@@ -12,51 +12,47 @@ limitations under the License.
 */
 
 import * as core from "@actions/core";
+import * as github from "@actions/github";
 import * as sigstore from "sigstore";
 import * as process from "process";
 import * as fs from "fs";
 import * as child_process from "child_process";
-
-interface githubObj {
-  event_name: string;
-  run_attempt: string;
-  run_id: string;
-  run_number: string;
-  workflow: string;
-  sha: string;
-  repository: string;
-  repository_owner: string;
-  // TODO(#1411): Record if these become available.
-  // repository_id: string;
-  // repository_owner_id: string;
-  // actor_id: string;
-  ref: string;
-  ref_type: string;
-  actor: string;
-}
+import { githubObj, rawTokenInterface, createPredicate } from "./predicate";
+import { getEnv, resolvePathInput } from "./utils";
 
 async function run(): Promise<void> {
   try {
-    /* Test locally:
+    /* Test locally. Requires a GitHub token:
         $ env INPUT_SLSA-WORKFLOW-RECIPIENT="delegator_generic_slsa3.yml" \
```

Comment:
nit: I'm not sure you need to use `env` but either way is fine 🤷 
```suggestion
        $ INPUT_SLSA-WORKFLOW-RECIPIENT="delegator_generic_slsa3.yml" \
```

Reply:
I had to, because vars really disliked having `-` characters.